lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
Yes. 

In fact what is going on here is sharing of properties between the loaders of different apps using the same namespace.

Which means I'm being dumb about what I'm suggesting with registerProperty. The same code running against the same namespace in different apps was one of the original goals here. So hmm. Don't use register property. Do write some inline documentation to explain why you are not using it.